Validator.py

```diff
@@ -0,0 +1,95 @@
+import glob
+import seaborn as sns
+from prompts import *
+
+class Validator:
+    def __init__(self, bad=None, confusion_matrix=None):
+        if bad is None:
+            bad = 0
+        self.bad = bad
+        if confusion_matrix is None:
+            confusion_matrix = [[0,0,0,0,0],
+                                [0,0,0,0,0],
+                                [0,0,0,0,0],
+                                [0,0,0,0,0],
+                                [0,0,0,0,0]] 
+        self.confusion_matrix = confusion_matrix
+    
+    def response_number(self, response):
```

Comment:
I ova funkcija je nepotrebno velika, pogledati ovu implementaciju:

def response_number(self, response):
    response_map = {
        "pie chart": 0,
        "bar chart": 1,
        "line graph": 2,
        "list": 3,
        "trash": 4
    }
    return response_map.get(response.lower(), 4 if not self.bad.__iadd__(1) else 4)